### PR TITLE
Toast to snackbar messages #77

### PR DESCRIPTION
**Pull request by git-typecode**
# Description

added id for activity_hotels layout.
added Snackbar in HotelsActivity.
removed Toast in HotelsActivity.

Fixes #(issue)
#77
## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: Android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Thu May 24 11:12:39 IST 2018
+#Tue Jul 03 11:13:27 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5-all.zip
```
Comment (Swati4star) [inline Android/gradle/wrapper/gradle-wrapper.properties]: It's still here.

Comment (git-typecode) [reply Android/gradle/wrapper/gradle-wrapper.properties]: isit? click on the file and look at it raw. its not there and the line is red with a (-) or do you mean both should be removed?

Comment (Swati4star) [reply Android/gradle/wrapper/gradle-wrapper.properties]: I am just saying that this change should not be here. 
So, you should remove this file from the commit

**Comment by Swati4star** [comment]
@git-typecode I am not asking you to remove distribution-url. I am asking you to not change anything in that.

**Comment by Swati4star** [comment]
@git-typecode Are you going to convert other toast to snackbar as a part of this PR only?

**Comment by git-typecode** [comment]
@Swati4star no, i was thinking that I want to open a new PR for the rest and keep this as is if that is OK.

**Pull request merged**